CLI/test/ressources/dependencies_1/output_expected/ProcedureCall-Public.rdz.tcbl

```diff
@@ -90,7 +90,7 @@
              OR TC-Callee-ebb060ea-Idt not = 'ebb060ea'
                PERFORM TC-LOAD-POINTERS-Callee
            END-IF
-      *    Equivalent to call ebb060eaMyPublicProcedure in module Callee
+      *    Equivalent to call ebb060ea in module Callee
```

Comment:
I think we should add a comment to clarify that the program ebb060ea is the generated code for function check here as well

Reply:
We have the original TypeCobol call commented above.
We can add this in #1331 which will remove the original TypeCobol call.

Reply:
Or simply replace the comment by 'Equivalent to call <TC-procedure-name> in module <module-name>', that is not mentioning the generated code in comment but only refering to TC names ?

Reply:
This comment is useful when you debug a TypeCobol program, because our tools will only show the Cobol program name (here : ebb060ea) and not the procedure name.